roles/repository/tasks/main.yml

```
@@ -11,17 +11,17 @@
 
     - name: Set solr.sharedSecret property
       set_fact:
-        temp_array: '{{ temp_array }} + [ ''-Dsolr.sharedSecret="{{ reposearch_shared_secret }}"'' ]'
+        temp_array: "{{ temp_array + [ '-Dsolr.sharedSecret=' ~ reposearch_shared_secret ] }}"
```

what if `reposearch_shared_secret` contains a space? I'm afraid it would break the command line.
Wouldn't this be more robust:

```suggestion
        temp_array: >-
           {{ temp_array + [ '-Dsolr.sharedSecret=' ~ reposearch_shared_secret | quote ] }}
```